index.html

```diff
@@ -258,12 +307,71 @@ <h3 class="title-1">
           </ul>
           <button class="see-proj">
-            <a class="project-link" href="">See project</a>
+            <a class="project-link" id="button-4" href="#">See project</a>
           </button>
         </div>
       </div>
     </section>
-    <section class="desk-works flex-column">
+    <div class="desk-pop desk-content-gone">
+      <div class="pop-content">
+        <h3 class=" pop-title"></h3>
+         <img class="pop-close" id="desk-pop-close" src="images/pop-close.svg" alt="">
+        
+        <ul class="pop-work-his">
+          <li class="live"></li>
+          <li>
+            <img
+              src="images/Counter.svg"
+              alt="bullet-point-counter"
+              srcset="" />
+          </li>
+          <li class="rest-li"></li>
+          <li>
+            <img
+              src="images/Counter.svg"
+              alt="bullet-point-counter"
+              srcset="" />
+          </li>
+          <li class="rest-li"></li>
+        </ul>
+        <img
+        class="card-sections works-img"
+        src=""
+        alt="project-screenshot" />
+        <div class="desk-pop-down">
+          <p class="pop-text">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. <br>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.  
+          </p>
+          <ul class="desk-pop-list">
+            <li>
+              <button class="desk-stack-button pop-button">
+                <a href="http://"> HTML</a>
+              </button>
+            </li>
+            <li>
+              <button class="desk-stack-button pop-button">
+                <a href="http://"> CSS</a>
+              </button>
+            </li>
+            <li>
+              <button class="desk-stack-button pop-button">
+                <a href="http://">JavaScript</a>
+              </button>
+            </li>
+          </ul>
+          <div class="pop-ref">
+            <button class="desk-see-proj desk-pop-button">
+              <a class="desk-project-link desk-pop-link" href="">See live <img src="images/see-live-logo.svg" alt=""></a>
+            </button>
+            <button class="desk-see-proj desk-pop-button">
+              <a class="desk-project-link desk-pop-link" href="">See source <img src="images/see-source-logo.svg" alt=""></a>
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="desk-works flex-column for-desktop">
       <div class="works-card">
```

Comment:
- [ ] [REQUIRED] Let's ensure that the desktop popup follows the design [template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). You need to work on the [X] button, it needs to be well-positioned closer to the edge of the div. Could you check the template provided?
- [ ] Also, the desktop popup is way too bigger for the screen when it pops. Kindly reduce the width a bit so it fits well on the screen when the `see project` button is been clicked.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="1183" alt="Screenshot 2023-04-12 at 11 16 55" src="https://user-images.githubusercontent.com/58692590/231431806-a35983ac-dd8b-4a45-976d-75eea52fc9d1.png"> | <img width="597" alt="Screenshot 2023-04-12 at 11 31 46" src="https://user-images.githubusercontent.com/58692590/231432000-a50a04d5-7dc5-4d54-8486-07b15eed348d.png">